src/scripts/Player.ts

```diff
@@ -851,6 +864,18 @@ class Player {
         this._questPoints(value);
     }
 
+    public ableToTravel() {
```

Comment:
These two methods may have a better place in the maphelper. At least that's where I would look for these methods

Reply:
Yeah, I wasn't 100% sure where they should go.